src/transformers/models/bert/modeling_bert.py

```diff
@@ -378,17 +375,13 @@ def forward(
         hidden_states,
         attention_mask=None,
         head_mask=None,
-        encoder_hidden_states=None,
-        encoder_attention_mask=None,
         past_key_value=None,
         output_attentions=False,
     ):
         self_outputs = self.self(
             hidden_states,
             attention_mask,
             head_mask,
-            encoder_hidden_states,
-            encoder_attention_mask,
```

Comment:
Why are we not passing the `encoder_hidden_states` to the self attention anymore?

Reply:
Think this was a typo, we obviously still need to pass the `encoder_hidden_states` for the cross-attention. However we don't need to pass the `encoder_attention_mask`. @LysandreJik for context, I asked @abhishekkrthakur to remove the `encoder_attention_mask` from his API of ConvBert here: https://github.com/huggingface/transformers/pull/9717#discussion_r564714433 and https://github.com/huggingface/transformers/pull/9717#pullrequestreview-576631779 . He then realized that we were doing this for Bert as well which is why the PR was opened . 

As explained previously, I think it's nice to not have two function args: `encoder_attention_mask` and `attention_mask` if only ever one of them is going to be used for the exact same purpose. Think it'll be nicer to only have `atteniton_mask`.

Reply:
I see, thank you for the context!